packages/wrangler/src/index.tsx

```diff
@@ -723,7 +723,7 @@ export async function main(argv: string[]): Promise<void> {
     () => {
       // "[DEPRECATED] 🦀 Build your project (if applicable)",
       throw new DeprecationError(
-        "`wrangler build` has been deprecated, please refer to https://github.com/cloudflare/wrangler2/blob/main/docs/deprecations.md#build for alternatives"
+        "`wrangler build` has been deprecated, please refer to https://github.com/cloudflare/cloudflare-docs/blob/production/content/workers/wrangler/migration/deprecations.md#build for alternatives"
```

Comment:
I think these URLs should be https://developers.cloudflare.com/workers/wrangler/migration/deprecations/

Reply:
Yeah I wasn't sure if we should link to the raw markdown files or the actual docs - will update

Reply:
If we are suggesting that they should update the file then I think the github link is better.
But here I think we are just pointing them to the information, right?
We used the Github link before because this info was not yet on the docs site.